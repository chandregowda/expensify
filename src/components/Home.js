import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const Home = () => (
	<div>
		<p>Home Page</p>
		<hr />
		<ExpenseListFilters />
		<ExpenseList />
	</div>
);

export default Home;
