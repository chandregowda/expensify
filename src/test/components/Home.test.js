import React from 'react';
import { shallow } from 'enzyme';
import Home from '../../components/Home';

test('should load home page', () => {
	const wrapper = shallow(<Home />);
	expect(wrapper).toMatchSnapshot();
});
