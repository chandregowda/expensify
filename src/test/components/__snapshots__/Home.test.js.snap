// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should load home page 1`] = `
<div>
  <p>
    Home Page
  </p>
  <hr />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
